### PR TITLE
Chore/add change email instructions

### PR DESCRIPTION
**Pull request by saltcod**
Add instructions for how to change your email 

https://www.notion.so/supabase/I-d-like-to-change-my-email-address-easily-4838ca283ce34abe8bd1d499a8677b8b?d=dbcd8461748e45d7941d1e072de5abc7
screenshot-2024-06-07-at-10 20 36

### CODE REVIEW

File: apps/studio/components/interfaces/Account/Preferences/AccountInformation.tsx [hunk 1]
```diff
@@ -17,21 +27,76 @@ const AccountInformation = ({ profile }: { profile?: Profile }) => {
...
+                        className="transition-transform duration-200"
+                        strokeWidth={1.5}
+                        size={14}
+                      />
+                    </div>
+                  </CollapsibleTrigger_Shadcn_>
+                  <CollapsibleContent_Shadcn_ className=" mt-2">
+                    <div className="bg-surface-200 p-4 rounded-lg grid gap-4">
+                      <div>
+                        <h4 className="text-sm">If you used GitHub to login: </h4>
```
Comment (mansueli) [inline apps/studio/components/interfaces/Account/Preferences/AccountInformation.tsx:69]: We should make this smarter by identifying the type of the session and showing the specific instructions like we do on line 100:

```     
     {session?.user.app_metadata.provider === 'email' && (
```

**Comment by mansueli** [comment]
LGTM

File: apps/studio/components/interfaces/Account/Preferences/AccountInformation.tsx [hunk 2]
```diff
@@ -17,21 +27,79 @@ const AccountInformation = ({ profile }: { profile?: Profile }) => {
...
+                disabled
+                placeholder="Email"
+                value={profile?.primary_email ?? ''}
+              />
+              <div className="col-span-8 mt-4">
+                <Collapsible_Shadcn_ className="-mt-1.5 pb-1.5">
+                  <CollapsibleTrigger_Shadcn_ className="group  font-normal p-0 [&[data-state=open]>div>svg]:!-rotate-180">
+                    <div className="flex items-center gap-x-2 w-full">
+                      <p className="text-xs text-foreground-light group-hover:text-foreground transition">
+                        Can I change my email?
```
Comment (ivasilov) [inline apps/studio/components/interfaces/Account/Preferences/AccountInformation.tsx:57]: I would rephrase this as `How can I change my email?`

File: apps/studio/components/interfaces/Account/Preferences/AccountInformation.tsx [hunk 3]
```diff
@@ -17,21 +27,79 @@ const AccountInformation = ({ profile }: { profile?: Profile }) => {
...
+                            <li>Invite this email to be an owner of the Org</li>
+                            <li>Join the Org</li>
+                            <li>Remove the old email from the Org</li>
+                            <li>Email Supabase Support to delete the old email (Optional) </li>
+                          </ul>
+                        </div>
+                      ) : (
+                        <div>
+                          <ul className="prose text-sm ml-4 mt-2">
+                            <li>Log out of Supabase</li>
```
Comment (ivasilov) [inline apps/studio/components/interfaces/Account/Preferences/AccountInformation.tsx:81]: Same here for `ol`.

**Review by ivasilov — APPROVED** [review]
